stable/jenkins/templates/_helpers.tpl

```diff
@@ -14,3 +14,12 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{- define "jenkins.kubernetes-version" -}}
+  {{- range .Values.Master.InstallPlugins -}}
+    {{ if hasPrefix "kubernetes:" . }}
```

Comment:
note that the entire name of the plugin including the colon should be used to avoid matching on other similarly named plugins. ex. `kubernetes:`